scripts/packages/VSCodeServer/src/eval.jl

```diff
@@ -96,6 +96,8 @@ function repl_runcode_request(conn, params::ReplRunCodeRequestParams)
                         @eval Main ans = $(QuoteNode(ans))
                     catch err
                         EvalError(err, catch_backtrace())
+                    finally
+                        JSONRPC.send_notification(conn_endpoint[], "repl/finisheval", nothing)
```

Comment:
Together with https://github.com/julia-vscode/julia-vscode/blob/master/src/interactive/repl.ts#L303 this will trigger the WS update twice, no?

Reply:
yes, so I removed those calls (see: https://github.com/julia-vscode/julia-vscode/blob/57755e25779c30a90a10cdc314cdbf1465507bfc/src/interactive/repl.ts#L259-L298 and https://github.com/julia-vscode/julia-vscode/blob/57755e25779c30a90a10cdc314cdbf1465507bfc/src/interactive/repl.ts#L353-L392)